config/settings/base.py

```diff
@@ -298,10 +296,9 @@
 USE_TZ = True
 
 # Static files (CSS, JavaScript, Images)
-STATIC_URL = f"https://{CORGI_DOMAIN}/"
+STATIC_URL = "/"
```

Comment:
is the intent to not force https scheme ... perhaps http->https is forced elsewhere in infra.